src/kibana/components/vislib/lib/data.js

```diff
@@ -289,6 +289,24 @@ define(function (require) {
     };
 
     /**
+     * Returns true if null values are present
+     * @returns {*}
+     */
+    Data.prototype.nullsPresent = function () {
+      var chartData = this.chartData();
+
+      return chartData.some(function (chart) {
```

Comment:
This is a lot simpler if you just use `some` on every level, like:

```
return chartData.some(function (chart) {
  return chart.series.some(function (obj) {
    return obj.values.some(function (d) {
      return d.y === null;
    });
  });
});
```

It'll also return faster, since it'll stop looking the first time it finds a null value instead of having to process the entire list.